src/app/views/home.cljs

```diff
@@ -120,3 +120,12 @@ Type `(start)` when you're ready!")
                       "sm:mb-0"
                       "mb-14"]}
         [repl/view]]])}))
+
+(defn- update-location-of-editor []
+  (let [window-width (. js/window -innerWidth)
+        location-of-editor-dom (. js/document getElementById "location-of-editor")]
+    (set! (. location-of-editor-dom -innerHTML)
+          (if (< window-width 640) "Down below" "Here on the right"))))
+
+((. js/window -addEventListener) "load" update-location-of-editor)
+((. js/window -addEventListener) "resize" update-location-of-editor)
```

Comment:
I just noticed that this code is calling `addEventListener` and `getElementById` in two different ways (using dash at the beginning (`-addEventListener`) and not using dash (`getElementById`))
It would be cool if they both used the same approach so the code would read the same.

Reply:
You can write this as:

```
(.addEventListener js/window "load" ...)
```